### PR TITLE
Consul

### PR DESCRIPTION
**Pull request by castroev**

### CODE REVIEW

File: ci-bootstrapper-job.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: ${bootstrap}-job
+  name: auditorbootstrap
```
Comment (JoeNunnelley) [inline ci-bootstrapper-job.yaml]: I prefer leaving this a varible if possible so it can be re-used

File: Jenkinsfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ pipeline {
     }
 
     environment {
-        repository = "tylerorg"
+        repository = "index.docker.io/tylerorg"
```
Comment (JoeNunnelley) [inline Jenkinsfile:10]: my PR and yours will clash here so whoever merges last will need to rebase changes in their branch before merging.

Comment (castroev) [reply Jenkinsfile]: I'll change it

**Comment by JoeNunnelley** [comment]
hm looks like it fails don't merge till it works

**Pull request merged**